bundles/org.openhab.binding.entsoe/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,177 @@
+					<option value="17Y000000930815H">FR_FR_IFA2 vhub_FR_IFA2</option>
+					<option value="17Y000000930816F">FR_IT_SAV_PIE vhub_IT_Savoie_Piemont</option>
+					<option value="17Y000000930817D">FR_FR_SAV_PIE vhub_FR_Savoie_Piemont</option>
+					<option value="17Y0000009369493">FR_UK_IFA2_V_BZN IFA2_virtual_BZN</option>
+					<option value="44Y-00000000160K">FI_FS Fingrid Oyj</option>
+					<option value="44Y-00000000161I">FI_EL Fingrid Oyj</option>
+					<option value="45Y000000000001C">DKW-NO2 DKW-NO2 virtual Bidding Zone Border</option>
+					<option value="45Y000000000002A">DKW-SE3 DKW-SE3 virtual Bidding Zone Border</option>
+					<option value="45Y0000000000038">DKW-DKE DKW-DKE virtual Bidding Zone Border</option>
+					<option value="46Y000000000001Y">SE3_FS SE3_FS (SE3, Fennoskan)</option>
+					<option value="46Y000000000002W">SE3_KS SE3_KS (SE3, Kontiskan)</option>
+					<option value="46Y000000000003U">SE4_SP SE4_SP (SE4, Swepol link)</option>
+					<option value="46Y000000000004S">SE4_NB SE4_NB (SE4, Nordbalt)</option>
+					<option value="46Y000000000005Q">SE4_BC SE4_BC (SE4, Baltic Cable)</option>
+					<option value="46Y000000000007M">CUT_AREA_SE3LS Cut area SE3LS</option>
+					<option value="46Y000000000008K">CUT_AREA_SE3 Cut area SE3</option>
+					<option value="46Y000000000009I">CUTCOR_SE3LS-SE3 Cut corridor SE3LS-SE3</option>
+					<option value="46Y000000000015N">VBZ_SE3-SE4_ACSW Virtual bidding zone border SE3-SE4 AC+SWL</option>
+					<option value="46Y000000000016L">VBZ_SE4-SE3_ACSW Virtual bidding zone border SE4-SE3 AC+SWL</option>
+					<option value="46Y000000000017J">SE3_SWL SE3_SWL (SE3, Sydvastlanken)</option>
+					<option value="46Y000000000018H">SE4_SWL SE4_SWL (SE4, Sydvastlanken)</option>
+					<option value="46Y000000000019F">CUTCOR_SE3A-SE3 Cut corridor SE3A-SE3</option>
+					<option value="50Y0JVU59B4JWQCU">NO_NO2NSL Virtual Bidding Zone NO2NSL</option>
+					<option value="50Y73EMZ34CQL9AJ">NO_NO2_NL Virtual Bidding Zone NO2-NL</option>
+					<option value="50YCUY85S1HH29EK">NO_NO2_DK1 Virtual Bidding Zone NO2-DK1</option>
+					<option value="50Y-HTS3792HUOAC">NO_NO2-GB Virtual bidding Zone NO2-GB</option>
+					<option value="50YNBFFTWZRAHA3P">NO_NO2-DE Virtual bidding Zone NO2-DE</option>
+					<option value="67Y-VISKOL-2003T">VISKOL2003 VISKOL DOO NOVI SAD SRBIJA</option>
+				</options>
+			</parameter>
+			<parameter name="historicDays" type="integer" min="1" max="365" required="false">
+				<label>Historic Data</label>
+				<default>1</default>
+				<description>Specify number of days to download historic data of energy spot prices (historic data will get exchange
+					rate of today)</description>
+			</parameter>
+			<parameter name="spotPricesAvailableUtcHour" type="integer" min="0" max="23" required="false">
+				<label>Publish Hour</label>
+				<default>11</default>
+				<description>UTC hour spot prices will get published on ENTSO-E. Usually at 11 UTC.</description>
```

Comment:
I see you have changed approach to use **Europe/Paris** time-zone. Since this was only a question, I just would like to be sure: Have you confirmed that UTC was wrong, and that it is indeed at 13:00 Paris time that prices will be published?